core/asset/service.go

```diff
@@ -19,6 +19,7 @@ type Service struct {
 	lineageRepository   LineageRepository
 	worker              Worker
 	logger              log.Logger
+	assetConfig         Config
```

Comment:
let's avoid unnecessary context, we can use config as name
```suggestion
	config         Config
```